### PR TITLE
v1.19.0 - styles for order card component

### PR DESCRIPTION
**Pull request by dandel10n**
### Added
- Styles for `c-orderCard`.

Layout can be checked in [this PR](https://github.com/justeat/global-component-library/pull/74)

Scrolling:
[Image from Gyazo](https://gyazo.com/cde2f54d00da1e470d9bfff7427c8a5c)

Hover:
[Image from Gyazo](https://gyazo.com/16ee583fd1b62ec477812382d747b471)

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,14 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html)
 
 
+v1.19.0
+------------------------------
+*December 10, 2018*
```
Comment (xander-marjoram) [inline CHANGELOG.md]: 📆 
Otherwise, looks good to me!

File: src/scss/components/optional/_previous-order-card.scss [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+        line-height: line-height(16px, 23px);
+    }
+
+    .c-previousOrderCard-orderTotal {
+        margin-right: spacing();
+        margin-left: spacing(x3);
+    }
+
+    .c-previousOrderCard-content {
+        padding: spacing() spacing(x2) spacing(x0.5) spacing(x2);
```
Comment (kevinrodrigues) [inline src/scss/components/optional/_previous-order-card.scss]: `spacing() spacing(x2) spacing(x0.5);`

File: src/scss/components/optional/_previous-order-card.scss [hunk 3]
```diff
@@ -0,0 +1,107 @@
...
+        color: $grey--mid;
+        width: 280px;
+        min-height: 259px;
+
+        @include media('>=mid') {
+            width: 392px;
+            min-height: 302px;
+        }
+
+        &:hover {
```
Comment (kevinrodrigues) [inline src/scss/components/optional/_previous-order-card.scss]: Should include a :focus.

File: src/scss/components/optional/_overflow-carousel.scss [hunk 4]
```diff
@@ -20,26 +20,61 @@
...
-                    overflow: visible;
-                }
+    .c-overflowCarousel-inner {
+        @include media('<mid') {
+            overflow-x: scroll;
+            overflow-y: hidden;
+            -webkit-overflow-scrolling: touch;
+            width: 100%;
+            &.is-active {
+
```
Comment (kevinrodrigues) [inline src/scss/components/optional/_overflow-carousel.scss]: minor but could you remove the line break

File: src/scss/components/optional/_previous-order-card.scss (hunk 2 above)
Comment (dandel10n) [reply src/scss/components/optional/_previous-order-card.scss]: ✅

File: src/scss/components/optional/_previous-order-card.scss (hunk 3 above)
Comment (dandel10n) [reply src/scss/components/optional/_previous-order-card.scss]: Yes, thanks, added ✅

File: src/scss/components/optional/_overflow-carousel.scss (hunk 4 above)
Comment (dandel10n) [reply src/scss/components/optional/_overflow-carousel.scss]: ✅

File: src/scss/components/optional/_previous-order-card.scss [hunk 5]
```diff
@@ -0,0 +1,108 @@
+/**
+* Components > Previous Order Card
+* =================================
+*
+* Example: Homepage – "Order it Again" block.
+*
+* The `c-previousOrderCard` component is an optional mixin within Fozzie.
```
Comment (ashleynolan) [inline src/scss/components/optional/_previous-order-card.scss]: Is it worth naming this as just an `orderCard`?  Just wondering if naming it like this means we couldn't use it elsewhere if we wanted to reuse the styles again?

**Pull request merged**